keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -1100,13 +1107,27 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason: `Bucket provisioning 'bucket-name' failed`
-- [ ] API .status
-  - Condition `PodReady=False "Error: secrets 'bucket-secret' not found"`) 
-  - Other field:
-- [ ] Other (treat as last resort)
-  - Details:
+- Bucket
+  - [ ] Events
+    - Event Reason: `Bucket provisioning 'bucket-name' failed`
+    - TODO: capture more Events
+  - [ ] API .status
+    - Condition `PodReady=False "Error: secrets 'bucket-secret' not found"`)
+    - TODO: investigate more
+- BucketAccess
+  - [ ] Events
+    - Event Reason: `BucketAccess provisioning 'bucket-access-name' failed`
+    - TODO: capture more Events
+  - [ ] API .status
+    - Condition `PodReady=False "Error: secrets 'bucket-secret' not found"`)
+    - TODO: investigate more
```

Comment:
@wlan0 we were a little unsure what is meant by the "Condition `PodReady=False "Error: secrets 'bucket-secret' not found"`)" text. We can discuss more next week unless you'd like to provide some more context here.